watch/core/rest/viewsets.py

```diff
@@ -0,0 +1,34 @@
+from django_filters.rest_framework import DjangoFilterBackend
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework.decorators import action
+from rgd.rest.base import ModelViewSet, ReadOnlyModelViewSet
+
+from .. import filters, models, serializers
+from ..tasks import jobs
+
+
+class RegionViewset(ModelViewSet):
+    serializer_class = serializers.RegionSerializer
+    # lookup_field = 'pk'
+    queryset = models.Region.objects.all()
+
+
+class GoogleCloudRecordViewset(ModelViewSet):
+    serializer_class = serializers.GoogleCloudRecordSerializer
+    # lookup_field = 'pk'
+    queryset = models.GoogleCloudRecord.objects.all()
+    filter_backends = [DjangoFilterBackend]
+    filterset_class = filters.GoogleCloudRecordFilter
+
+    # @swagger_auto_schema(
+    #     method='POST',
+    #     operation_summary='Ingest records using search params.',
+    # )
+    # @action(detail=True)
```

Comment:
```suggestion
    # @action(detail=True, methods=['post'])
```

Try this to do a `POST`.